biom/table.py

```diff
@@ -611,6 +639,85 @@ def add_group_metadata(self, group_md, axis='sample'):
         else:
             raise UnknownAxisError(axis)
 
+    def del_metadata(self, keys=None, axis=None, inplace=False):
```

Comment:
Can you rename the test method to indicate its intention, or split it out into two methods? It's not generally good practice to test more than one thing in a unit test.